tf-repo-mgmt/repository_sync/modules/github/github.repository.environment.copilot.tf

```diff
@@ -12,18 +12,18 @@ resource "github_actions_environment_secret" "copilot_tenant_id" {
   plaintext_value = var.arm_tenant_id
 }
 
-resource "github_actions_environment_secret" "copilot_subscription_id" {
+resource "github_actions_environment_secret" "copilot_client_id" {
   count           = var.repository_creation_mode_enabled ? 0 : 1
   repository      = github_repository.this.name
   environment     = github_repository_environment.copilot[0].environment
-  secret_name     = "ARM_SUBSCRIPTION_ID"
-  plaintext_value = var.arm_subscription_id
+  secret_name     = "ARM_CLIENT_ID"
+  plaintext_value = var.arm_client_id
 }
 
-resource "github_actions_environment_secret" "copilot_client_id" {
+resource "github_actions_environment_secret" "copilot_subscription_id" {
   count           = var.repository_creation_mode_enabled ? 0 : 1
   repository      = github_repository.this.name
-  environment     = github_repository_environment.copilot[0].environment
-  secret_name     = "ARM_CLIENT_ID"
```

Comment:
why this change needed?
2 clock just exchange places in a file

Reply:
No reason. This PR is still work in progress.